index.html

```diff
@@ -165,6 +165,21 @@ <h2>Skills</h2>
         </li>
       </ul>
       </section>
+      <section class="form">
```

Comment:
- [ ] [OPTIONAL] As you can see there is some space around your website that is making the design ugly.  You can fix it by  adding this code to the body in CSS `margin: 0`

Your design
![image](https://user-images.githubusercontent.com/69011963/172436336-2d31dc82-9a62-47e0-a888-3af40e374ed6.png)